Tests/SWBBuildSystemTests/LinkerTests.swift

```diff
@@ -229,22 +229,33 @@ fileprivate struct LinkerTests: CoreBasedTests {
                 """
             }
 
+            // // Try to find the installed linkers
+            let ldLinkerPath = try? await self.ldPath
```

Comment:
I'd leave the `try?` as `try` -- if this throws, the Core failed to initialize, and the whole test is bogus.